executor_manager/tests/executors/test_base.py

```
@@ -16,87 +17,128 @@ def test_executor_is_abstract(self):
+            def get_executor_count(self, label_selector=None):
+                pass
+
         with pytest.raises(TypeError):
             IncompleteExecutor()
 
     def test_complete_executor_implementation(self):
         """Test that a complete implementation can be instantiated"""
-        
+
         class CompleteExecutor(Executor):
             def submit_executor(self, task, callback=None):
```

Ruff (0.14.13)

Unused method argument: `task`

(ARG002)